src/requirements/data/majors/psych.ts

```diff
@@ -0,0 +1,130 @@
+
+const psychRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Credit Requirements',
+    description:
+      'A total of 40 credits in psychology (including prerequisites), ' +
+      'from which students majoring in psychology are expected to choose, in consultation with their advisors, ' +
+      'a range of courses that covers the basic processes in psychology.',
+    source: 'https://psychology.cornell.edu/major',
+    checker: includesWithSingleRequirement('PSYCH ****'),
+    fulfilledBy: 'credits',
+    perSlotMinCount: [40],
+    allowCourseDoubleCounting: true,
+    disallowTransferCredit: true,
+  },
+  {
+    name: 'Statistics Requirements',
+    description: 'Students are required to demonstrate their proficiency in statistics',
+    source: 'https://psychology.cornell.edu/major',
+    checker: includesWithSubRequirements([
+      'PSYCH 2500',
+      'ILRST 2100',
+      'STSCI 2100',
+      'BTRY 3010',
+      'STSCI 2200',
+      'NTRES 3130',
+      'PAM 2100',
+      'ILRST 3100',
+      'STSCI 3110',
+      'ECON 3110',
+      'ECON 3130',
+      'AEM 2100',
+      'STSCI 2150',
+      'SCO 3010',
+    ]),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Statistics'],
+  },
+  {
+    name: 'General Requirements',
+    description:
+      'Unless otherwise approved by the advisor, all Psychology students should take ' +
+      'at least one course in each of the following three areas of psychology.',
+    source: 'https://psychology.cornell.edu/major',
+    checker: includesWithSubRequirements(
+      [
+        'PSYCH 1102',
+        'PSYCH 2050',
+        'PSYCH 2090',
+        'PSYCH 2150',
+        'PSYCH 3140',
+        'PSYCH 3160',
+        'PSYCH 3350',
+        'PSYCH 3420',
+        'PSYCH 4120',
+        'PSYCH 4180',
+        'PSYCH 4270',
+        'PSYCH 4280',
+        'PSYCH 4331',
+        'PSYCH 4320',
+        'PSYCH 4360',
+        'PSYCH 4370',
+        'PSYCH 4650',
+        'PSYCH 4770',
+        'PSYCH 4780',
+      ],
+      [
+        'PSYCH 2230',
+        'PSYCH 3220',
+        'PSYCH 3240',
+        'PSYCH 3260',
+        'PSYCH 3320',
+        'PSYCH 4200',
+        'PSYCH 4230',
+        'PSYCH 4240',
+        'PSYCH 4250',
+        'PSYCH 4260',
+        'PSYCH 4350',
+        'PSYCH 4390',
+        'PSYCH 4400',
+        'PSYCH 4410',
+      ],
+      [
+        'PSYCH 1500',
+        'PSYCH 2650',
+        'PSYCH 2750',
+        'PSYCH 2800',
+        'PSYCH 2820',
+        'PSYCH 2940',
+        'PSYCH 3250',
+        'PSYCH 3270',
+        'PSYCH 3280',
+        'PSYCH 3800',
+        'PSYCH 3850',
+        'PSYCH 4050',
+        'PSYCH 4300',
+        'PSYCH 4810',
+        'PSYCH 4820',
+        'PSYCH 4840',
+        'PSYCH 4850',
+        'PSYCH 4910',
+      ],
+      ['PSYCH 1101', 'PSYCH 1650', 'PSYCH 1990', 'PSYCH 2500', 'PSYCH 4700', 'PSYCH 4710']
```

Comment:
I don't understand the purpose of the fourth group since the minimum required is 0. Also, it looks to not even show up in the requirements sidebar at all, which seems like it could be a bug. I think that we would want to still allow users to add courses (potentially unlimited?) from the sidebar for slots that have a minimum of 0. Maybe this could be a special case, because with our current implementation, a 0 minimum seems to be useless.
<img width="357" alt="image" src="https://user-images.githubusercontent.com/47431797/187579260-11a8e992-1415-42e9-ac7f-00c4e48dc461.png">

Reply:
I agree, I think they should be strictly opt-in.

Reply:
Is it also worth considering handling a 0 min in general, or should we just always use opt-in for that?

Reply:
I think just using opt-in in fine.

Reply:
For this req data, I think we can just remove those "other"s.

Reply:
I don't think it would ever make sense to intentionally put 0 min, since then it means the slot is always automatically fulfilled. Maybe it makes sense to handle that edge case in the case of bad data on our part, or programmatically block this kind of data.